docs/workflows/transitions/preprocess/preprocess/make_predictions.py

```diff
@@ -12,24 +12,43 @@ def required_labels(field_config):
     return set([label for label in field_config if field_config[label].get('required', True)])
 
 
+def is_line(field_config, p):
+    return field_config[p['label']]['type'] == 'lines'
+
+
 def filter_optional_fields(predictions, field_config):
     def predicate(p):
         conf_threshold = field_config[p['label']]['confidenceLevels']['low']
+        if is_line(field_config, p):
+            return True
         return p['label'] in required_labels(field_config) or conf_threshold < p['confidence']
     
     return list(filter(predicate, predictions))
 
 
 def filter_by_top1(predictions):
     labels = set(map(lambda p: p['label'], predictions))
-    
+
     def top1(label):
         preds = filter(lambda f: f['label'] == label, predictions)
+
         return max(preds, key=lambda p: p['confidence'])
 
     return [top1(label) for label in labels]
 
 
+def merge_predictions_and_gt(predictions, old_ground_truth):
+    old_ground_truth = {gt['label']: gt['value'] for gt in old_ground_truth}
+
+    # override value if label is the same, add if it is not predicted
+    for prediction in predictions:
+        prediction['value'] = old_ground_truth.pop(prediction['label'], prediction['value'])
```

Comment:
If we override a prediction with a ground truth, should we also adjust the confidence?